src/esp/bindings_js/modules/ar_utils.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+/* global Module */
+
+export const VIEW_SENSOR = "eye";
+
+export function getEyeSensorSpecs(resolutionWidth, resolutionHeight) {
```

Comment:
"AR" seems close enough to "XR" that this could go in xr_utils.js, and then we could omit this file.

Reply:
I think that would be too confusing. xr_utils contains stuff that is used for both VR and AR applications. But the functions in this file are specific to AR only.